docs/content/workflows/design/hooks/kafka-hook.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "Kafka hook"
+weight = 3
+
++++
+
+You want to run a workflow from a kafka message ? This kind of hook is for you.
```

Comment:
You want to run a workflow from a kafka message? This kind of hook is for you.

Reply:
s/kind/type

Reply:
done